style.css

```diff
@@ -116,3 +124,282 @@ header {
   justify-content: space-evenly;
   align-items: center;
 }
+
+.action-button {
+  background: #36b37f;
+  color: white;
+  border-radius: 4px;
+}
+
+.action-button:hover {
+  cursor: pointer;
+  background: white;
+  color: green;
+}
+
+@media screen and (max-width: 412px) {
+  body {
+    all: unset;
+  }
+
+  .work-section {
+    margin-top: 10px;
+  }
+
+  .opening-heading {
+    min-width: 327px;
+    height: 97px;
+  }
+
+  .work-heading {
+    text-align: center;
+    color: #172b4d;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .first-work {
+    list-style: none;
+    height: 451px;
+    width: 250px;
+    margin: 0 auto;
+    border: #d0d9d4 solid 1px;
+  }
+
+  .container-grid {
+    display: grid;
+    grid-template-rows: auto auto auto;
+
+    /* background-color: #2196F3; */
+    grid-row-gap: 25px;
+  }
```

Comment:
- [ ] Great work The main container of the projects has a display:grid. Kindly, remove the commented line